package.json

```diff
@@ -22,13 +22,12 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@laserfiche/lf-js-utils": "^3.0.5",
+    "@laserfiche/lf-js-utils": "^4.0.2",
```

Comment:
hmm I guess if we already had these dependencies we could have the types package as an actual dependency instead of peer.